sdl_android/src/androidTest/java/com/smartdevicelink/test/util/DeviceUtil.java

```diff
@@ -1,14 +1,18 @@
 package com.smartdevicelink.test.util;
 
 import android.os.Build;
+import android.util.Log;
```

Comment:
do we need this import?

Reply:
No. I will remove. Also have not tagged for review yet. ;)